PowerUp/app/src/main/java/powerup/systers/com/FinalAvatarActivity.java

```diff
@@ -89,12 +114,9 @@ public void onCreate(Bundle savedInstanceState) {
         continueButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(FinalAvatarActivity.this);
-                boolean hasPreviouslyStarted = prefs.getBoolean(getString(R.string.preferences_has_previously_started), false);

```

Comment:
@manishakeim @manjumegh @niksj I've shifted it to PrefHelper, please suggest if any changes needed.

Reply:
No, changes aren't required. It looks fine this way.